doc/index.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN""http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
This file is generated. It should not be added in the commit. Could you please remove it?